examples/snowflake/.gitignore

```diff
@@ -1 +0,0 @@
-Config.toml
```

Comment:
`Config.toml` was added to gitignore because it can contain credentials to Snowflake and to avoid the risk of accidentally pushing these credentials online. Would it be okay to remove this?

Reply:
I will add it back again.

Reply:
Fixed in 8f85f19